src/Assetic/Filter/CJSMinFilter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Assetic\Filter;
+
+use Assetic\Asset\AssetInterface;
+use Assetic\Exception\FilterException;
+
+/**
+ * Runs assets through the C jsmin binary.
+ *
+ * @link http://javascript.crockford.com/jsmin.html
+ * @author Felix Yeung
+ */
+class CJSMinFilter implements FilterInterface
+{
+    private $jsminBin;
+
+    public function __construct($jsminBin = '/usr/bin/jsmin')
+    {
+        $this->jsminBin = $jsminBin;
+    }
+
+    public function filterLoad(AssetInterface $asset)
+    {
+    }
+
+    public function filterDump(AssetInterface $asset)
+    {
+        $assetFile = $asset->getSourceRoot() . '/' . $asset->getSourcePath();
```

Comment:
this is wrong. You need to apply the filter on the current content of the asset, not on the original content. Otherwise you cannot chain filters.